### DISCUSSION TITLE
Building application

### DISCUSSION THREAD

**Discussion by Anthael** [jellyfin/jellyfin-androidtv#893] [General]
Hi,

According the procedure to compile, i have tried to build the last version for my tv philips one ( android pie ).

The application refused to complete the installation process but the github official release 11.5 succeded on installation.

Please, could you tell how to know why the application compiled refused to be installed ? 

And maybe I got something wrong on my installation.
Im using debian version, with android command line (instead of android studio).

Thanks by advance.
[build.txt](https://github.com/jellyfin/jellyfin-androidtv/files/6489685/build.txt)

**Comment by nielsvanvelzen** [comment]
If you've installed the debug variant of the 0.11.x version you need to uninstall it before installing 0.12.x because the version code is lowered (to 0). No need to do that again after 0.12 is released though :)

**Reply by Anthael** [reply]
thanks by i have already uninstall the 0.11.x before.
but the compiled debug version works but not the release one.

**Reply by nielsvanvelzen** [reply]
That's correct, the (unsigned-)release variant is not meant to be installed. When building manually you should install the debug version.

You should think of the "debug" variant as a "development" edition. It's pretty much the same as release except for Android allowing the installation unsigned.

**Reply by Anthael** [reply]
ok thanks

**Reply by thomux79** [reply]
Can anybody help me to add to apk file my server url. not each time on new device to fill it. Is it possible to make that URL added by default. Thanks for any help in advance
